src/inmanta/env.py

```diff
@@ -150,6 +151,19 @@ def activate_that(self) -> None:
         # Also set the python path environment variable for any subprocess
         os.environ["PYTHONPATH"] = os.pathsep.join(sys.path)
 
+        # write out a "stub" pip so that pip list works in the virtual env
+        bin_path = os.path.join(self.env_path, "bin", "pip")
+        with open(bin_path, "w") as fd:
+            fd.write(
+                f"""#!/bin/sh
+source activate
```

Comment:
this will only work if the script is executed in the folder it is in (./pip)   and not if the users cwd is any other folder (I think)

Reply:
you are wrong. I tested it. This is only the case when you do "source ./activate". Bash is an ugly beast